content/en/docs/concepts/cluster-administration/networking.md

```diff
@@ -293,6 +293,120 @@ Weave Net runs as a [CNI plug-in](https://www.weave.works/docs/net/latest/cni-pl
 or stand-alone.  In either version, it doesn't require any configuration or extra code
 to run, and in both cases, the network provides one IP address per pod - as is standard for Kubernetes.
 
+## IPv4/IPv6 dual stack
+
+{{< feature-state state="alpha" >}}
+
+If you enable IPv4/IPv6 dual stack networking for you Kubernetes cluster, the cluster will support the simultaneous assignment of both IPv4 and IPv6 addresses.
+
+### Supported Features
+
+Enabling IPv4/IPv6 dual stack on your Kubernetes cluster provides the following features:
+
+   * Dual stack Pod networking (a single IPv4 and IPv6 address assignment per Pod)
+   * IPv4 and IPv6 enabled Services (must be only of a single address family)
+   * Kubenet multi address family support (IPv4 and IPv6)
+   * Pod off-cluster egress routing (eg. the Internet) via both IPv4 and IPv6 interfaces
+
+### Prerequisites
+
+The following prerequisites are needed in order to utilize IPv4/IPv6 dual stack Kubernetes clusters:
+
+   * Kubernetes 1.16 or later
+   * Provider support for dual stack networking (Cloud provider or otherwise must be able to provide Kubernetes nodes with routable IPv4/IPv6 network interfaces)
+   * Kubenet network plugin
+   * Kube-proxy running in mode IPVS
+
+### Enable IPv4/IPv6 dual stack
+
+To enable IPv4/IPv6 dual stack, enable the `IPv6DualStack` [feature gates](https://kubernetes.io/docs/reference/command-line-tools-reference/feature-gates/) for the relevant components of your cluster, and set dual-stack cluster network assignments:
+
+   * kube-controller-manager:
+      * `--feature-gates="IPv6DualStack=true"`
+      * `--cluster-cidr=<IPv4 CIDR>,<IPv6 CIDR>` eg. `--cluster-cidr=10.244.0.0/16,fc00::/24`
+      * `--service-cluster-ip-range=<IPv4 CIDR>,<IPv6 CIDR>`
+   * kubelet:
+      * `--feature-gates="IPv6DualStack=true"`
+   * kube-proxy:
+      * `--proxy-mode=ipvs`
+      * `--cluster-cidrs=<IPv4 CIDR>,<IPv6 CIDR>` 
+      * `--feature-gates="IPv6DualStack=true"`
+
+{{< note >}}
+An IPv6 CIDR specificed via the `--cluster-cidr` flag larger than /24 will fail
+{{< /note >}}
+
+### Services
```

Comment:
Users need to know that they only need to configure services cidr range for dualstackness if they plan to ingress on both `ipv4` and ipv6` services. it is optional a cluster can be dualstack egress only.